BuildDevint

```diff
@@ -44,15 +44,15 @@ if [ $INTELLIJ_VERSION == "true" ] ; then
     ./tools/IntellijVersionHelper 2017.1
 fi
 
-./gradlew clean buildPlugin --project-dir ./PluginsAndFeatures/azure-toolkit-for-intellij -s -Pintellij_version=IC-2017.1 -Pdep_plugins=org.intellij.scala:2017.1.15
+./gradlew clean buildPlugin test --debug --project-dir ./PluginsAndFeatures/azure-toolkit-for-intellij -s -Pintellij_version=IC-2017.1 -Pdep_plugins=org.intellij.scala:2017.1.15
```

Comment:
what's `--debug` for? why there's no need for version `2017.2` ?

Reply:
I added this to see detailed log and the case always failed on Jenkins. It will block the build process. So the version 2017.2 never been executed. And after add --debug the log will be very long and the travis-ci will fail.